### PR TITLE
[bitnami/kibana] Add custom labels option to the service in Kibana chart

### LINKED ISSUE #8044: [bitnami/kibana] Add custom labels to the service in Kibana chart
**Which chart**:
bitnami/kibana

**Is your feature request related to a problem? Please describe.**
Adding custom labels to Kibana service is not possible as there is only the common labels are applied while creating the release.

**Describe the solution you'd like**
In Kibana service config values, expose another param named 'labels' and add those label after common labels are added to the service.

### PR DESCRIPTION
**Pull request by abdullah7**
**Description of the change**
Add custom labels value option to the service in Kibana chart. It sets additional labels to the service created by the Kibana release deployment.

**Benefits**
Being able to set additional labels which enables selectors to target the service for example a NetworkPolicy.

**Possible drawbacks**
Can not think of any!?

**Applicable issues**

  - fixes #8044

**Additional information**

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. [bitnami/<name_of_the_chart>])

### CODE REVIEW

File: bitnami/kibana/templates/service.yaml [hunk 1]
```diff
@@ -3,6 +3,9 @@ kind: Service
 metadata:
   name: {{ include "common.names.fullname" . }}
   labels: {{- include "common.labels.standard" . | nindent 4 }}
+{{- if .Values.service.labels }}
+{{ toYaml .Values.service.labels | indent 4}}
+{{- end }}
```
Comment (juan131) [inline bitnami/kibana/templates/service.yaml:8]: Please user the "common.tplvalues.render" named template instead: 
```suggestion
  {{- if .Values.service.labels }}
  {{- include "common.tplvalues.render" (dict "value" .Values.service.labels "context" $) | nindent 4 }}
  {{- end }}
```

**Review by juan131 — CHANGES_REQUESTED** [review]
Thanks so much for your contribution! Just a very minor request and we'll be ready to merger your changes

File: bitnami/kibana/templates/service.yaml (hunk 1 above)
Comment (abdullah7) [reply bitnami/kibana/templates/service.yaml]: Thanks for the suggestion! Already committed the changes.

File: bitnami/kibana/README.md [hunk 2]
```diff
@@ -125,6 +125,7 @@ The command removes all the Kubernetes components associated with the chart and
 | `service.nodePort`                     | Specify the nodePort value for the LoadBalancer and NodePort service types                                                                                | `""`                     |
 | `service.externalTrafficPolicy`        | Enable client source IP preservation                                                                                                                      | `Cluster`                |
 | `service.annotations`                  | Annotations for Kibana service (evaluated as a template)                                                                                                  | `{}`                     |
+| `service.labels`                  | Labels for Kibana service                                                                                                  | `{}`                     |
```
Comment (juan131) [inline bitnami/kibana/README.md:128]: Did you update the table using the [readme-generator tool](https://github.com/bitnami-labs/readme-generator-for-helm)? It looks like it's not correctly aligned. Could you take a look?

Comment (abdullah7) [reply bitnami/kibana/README.md]: Before it was not generated by that tool. I've just updated it through above mentioned readme-generator.

**Comment by rafariossaa** [comment]
Hi, thanks for fixing it.

LGTM.
I am merging it.

**Pull request merged**